sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.netstandard2.0.cs

```diff
@@ -1157,6 +1154,65 @@ public partial class NetAppVolumeGroupMetadata
         public string GroupDescription { get { throw null; } set { } }
         public long? VolumesCount { get { throw null; } }
     }
+    public partial class NetAppVolumeGroupResult : Azure.ResourceManager.Models.ResourceData
+    {
+        internal NetAppVolumeGroupResult() { }
+        public Azure.ResourceManager.NetApp.Models.NetAppVolumeGroupMetadata GroupMetaData { get { throw null; } }
+        public Azure.Core.AzureLocation? Location { get { throw null; } }
+        public string ProvisioningState { get { throw null; } }
+    }
+    public partial class NetAppVolumeGroupVolume
+    {
+        public NetAppVolumeGroupVolume(string creationToken, long usageThreshold, Azure.Core.ResourceIdentifier subnetId) { }
+        public Azure.ResourceManager.NetApp.Models.NetAppAvsDataStore? AvsDataStore { get { throw null; } set { } }
+        public System.Guid? BackupId { get { throw null; } set { } }
+        public string BaremetalTenantId { get { throw null; } }
+        public Azure.Core.ResourceIdentifier CapacityPoolResourceId { get { throw null; } set { } }
+        public int? CloneProgress { get { throw null; } }
+        public bool? CoolAccess { get { throw null; } set { } }
```

Comment:
Please check the name of this property in the original NetAppVolumeGroupData

Reply:
fixed